bower.json

```diff
@@ -30,7 +30,8 @@
     "angular": "1.3.15",
     "angular-route": "1.3.15",
     "angular-bootstrap": "0.13.0",
-    "datatables": "1.10.8"
+    "datatables": "1.10.8",
+    "jquery-form": "^3.46.0"
```

Comment:
FWIW, I usually pin exact versions rather than "allow updates to minor versions" since I'm paranoid about 3.46.1 coming out and being backwards-incompatible, but in general that's mostly paranoia and the alleged purpose of the ``^`` specifier is to say "only backwards-compatible changes", so it's probably fine as is.  Note that all of the other dependencies are pinned to exact versions for this reason though.